src/adaptors/utils.js

```diff
@@ -513,3 +513,83 @@ exports.getTotalSupply = async (tokenMintAddress) => {
 
   return supplyInTokens;
 };
+
+const merklNetworks = {
+  1: 'ethereum',
+  137: 'polygon',
+  10: 'optimism',
+  42161: 'arbitrum',
+  1101: 'polygon_zkevm',
+  8453: 'base',
+  60808: 'bob',
+  146: 'sonic',
+  43114: 'avax',
+  80094: 'berachain',
+  56: 'bsc',
+}
```

Comment:
Do you think I should import that from the merkl adapter?
maybe expose it as a separate file?